patches/server/0008-Paper-Metrics.patch

```diff
@@ -13,39 +13,53 @@ tangible effect on server performance. The data is used to allow the authors of
 PaperMC to track version and platform usage so that we can make better management
 decisions on behalf of the project.
 
+diff --git a/build.gradle.kts b/build.gradle.kts
+index 4db0cc3f8505747e77d314320545eb71904b4eac..6baabc30a363d132ea3d8a7da54a40aaf918f15b 100644
+--- a/build.gradle.kts
++++ b/build.gradle.kts
+@@ -16,6 +16,7 @@ dependencies {
+     implementation("org.apache.logging.log4j:log4j-iostreams:2.14.1") // Paper
+     implementation("org.ow2.asm:asm:9.2")
+     implementation("org.ow2.asm:asm-commons:9.2") // Paper - ASM event executor generation
++    implementation("org.bstats:bstats-base:2.2.1") // Paper
+     runtimeOnly("org.xerial:sqlite-jdbc:3.36.0.3")
+     runtimeOnly("mysql:mysql-connector-java:8.0.27")
+ 
+@@ -65,6 +66,7 @@ relocation {
+     relocate("org.bukkit.craftbukkit" to "org.bukkit.craftbukkit.v$packageVersion") {
+         exclude("org.bukkit.craftbukkit.Main*")
+     }
++    relocate("org.bstats:bstats-base" to "org.bstats") // Paper
```

Comment:
we don't relocate libs anymore right?

Reply:
[bStats requires you to relocate it](https://github.com/Bastian/bStats-Metrics/blob/master/base/src/main/java/org/bstats/MetricsBase.java#L211)